### ISSUE TITLE
Empty folder not synchronized because it is "too big"

### ISSUE THREAD

**Issue by PVince81** [owncloud/client#6084]
### Steps

1. Create a folder "test"
2. Put a good gigabyte of files into it
3. Setup local desktop client
4. Wait for sync
5. Desktop client warns that the folder "test" will not be synced because too big. Do not make any change here and leave the warning!
6. Stop sync client
7. In the web UI or different client, empty the folder "test"
8. Start sync client again

### Expected
Client syncs folder again as it's empty

### Actual
Desktop Client still shows the warning, the list displays the folder "test" in red with 0 bytes size.

### Versions
ownCloud 10.0.3
owncloud-client-2.3.3~rc2-8238.1.x86_64

@guruz @ogoffart minor bug

**Comment by ckamm** [comment]
I don't think we re-check the folder size once it's been put on the list of undecided-and-not-synced folders.

**Comment by PVince81** [comment]
I'll leave it up to you guys to decide if this is by design or not.

**Comment by ogoffart** [comment]
The thing is that the sync engine has decided it should be confirmed and stored this decision. Now the user has to accept that with the UI.  It does not matter if the folder has changed size.

I'm tempted to close this issue as a wontfix. any objection?

Otherwise should the UI automatically accepts folder that are now smaller that the threshold?  (because the user changed the threshold, or because the files were deleted later)

**Comment by PVince81** [comment]
Maybe @pmaier1 has other ideas about expected behavior ?
